x/evm/keeper/migrate.go

```diff
@@ -43,7 +45,27 @@ func migrateCmdType(ctx sdk.Context, ck chainKeeper, key key.Key, cmd types.Comm
 		return fmt.Errorf("command type %s is invalid at key %s", cmdType, key.String())
 	}
 	cmd.Type = types.CommandType(typeEnum)
-	return ck.getStore(ctx).SetNewValidated(key, &cmd)
+
+	// exclude param decoding from validation because legacy commands break. Need to clean up those commands in a future release
```

Comment:
can you add some logs around how many commands are migrated, command type being migrated etc. (can be a debug log to avoid spam)